util/package.go

```diff
@@ -68,11 +68,11 @@ type Image struct {
 }
 
 type DataSet struct {
-	Title          string                   `config:"title" json:"title"`
+	Title          string                   `config:"title" json:"title" validate:"required"`
 	Name           string                   `config:"name" json:"name"`
 	Release        string                   `config:"release" json:"release"`
-	Type           string                   `config:"type" json:"type"`
-	IngestPipeline string                   `config:"ingest_pipeline" config:"ingest_pipeline" json:"ingest_pipeline"`
+	Type           string                   `config:"type" json:"type" validate:"required"`
+	IngestPipeline string                   `config:"ingest_pipeline,omitempty" config:"ingest_pipeline" json:"ingest_pipeline,omitempty"`
```

Comment:
We'll need to update the `Dataset` type in EPM so [`ingest_pipeline`](https://github.com/elastic/kibana/blob/f51a5a935161358d53761c2b651996874e4e336d/x-pack/legacy/plugins/epm/common/types.ts#L123) is optional `ingest_pipeline?`

Reply:
@jfsiii Good catch, can you take care of this?